vital/types/local_geo_cs.cxx

```diff
@@ -62,7 +62,7 @@ write_local_geo_cs_to_file(local_geo_cs const& lgcs,
   kwiver::vital::logger_handle_t logger(kwiver::vital::get_logger(
     "write_local_geo_cs_to_file"));
   // write out the origin of the local coordinate system
-  auto lon_lat_alt = lgcs.origin().location( SRID::lat_lon_WGS84 );
```

Comment:
This explicit ` SRID::lat_lon_WGS84` is intentional.  It says "give me the location of the origin in WGS84 coordinate".  This is to ensure that the local_geo_cs file is always in lat, lon, alt (WGS84).  By default the origin often is in WGS84, so your change will still work as expected.  However, it is possible to create an origin point in a different coordinate system, so this conversion is necessary.